backend_python/services/vector_db.py

```
@@ -8,8 +8,30 @@
-from chromadb.config import Settings
+# Patch ChromaDB telemetry to suppress capture() errors
+try:
+    import chromadb.telemetry.client as telemetry_client
+    if hasattr(telemetry_client, 'TelemetryClient'):
+        original_capture = getattr(telemetry_client.TelemetryClient, 'capture', None)
+        if original_capture and callable(original_capture):
+            def safe_capture(self, *args, **kwargs):
+                """Suppress telemetry capture errors."""
+                try:
+                    return original_capture(self, *args, **kwargs)
+                except (TypeError, Exception) as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)